src/cogent3/app/sample.py

```diff
@@ -820,18 +820,46 @@ def main(self, seqs: SeqsCollectionType) -> T:
 class trim_stop_codons:
     """Removes terminal stop codons."""
 
-    def __init__(self, gc=1):
+    def __init__(self, gc: Union[str, int] = 1):
         """
         Parameters
         ----------
         gc
             identifier for a genetic code or a genetic code instance, defaults
-            to standard genetic code
+            to standard genetic code. See https://cogent3.org/doc/cookbook/what_codes.html
 
         Returns
         -------
         A new sequence collection, or False if not all the named sequences are
         in the collection.
+
+        Examples
+        --------
+
+        Removes trailing (terminal) stop codons from sequences in a sequence
+        collection or alignment.
+
+        Create a sample unaligned sequence data and an app to remove trailing
+        stop codons.
+
+        >>> from cogent3 import make_unaligned_seqs, make_aligned_seqs, get_app
+        >>> ualn = make_unaligned_seqs(data={"s1": "AAATTTCCC", "s2": "AAATTTTAA"}, moltype="dna")
+        >>> app = get_app("trim_stop_codons")
+        >>> result = app(ualn)
+        >>> print(result.to_dict())
```

Comment:
change to pretty_print()

Reply:
`AttributeError: 'SequenceCollection' object has no attribute 'to_pretty'`. Works for the next example where it's aligned. Suggestions?

Reply:
`AttributeError: 'SequenceCollection' object has no attribute 'to_pretty'`. Works for the next example that returns an `Aligned` instance. Suggestions?

Reply:
Nah, you're right. `to_dict()` is fine.